src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,12 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (!this.store) {
+                this.store = window.storeService.open(window.name);
+            }
+            if (!this.store.isCompact() && stock.favourite) {
+                this.selectionService.select(stock);
+            }
```

Comment:
I kind of see it like bookmarks. The favourites are the stocks you care about most, and you get to see the sparkline preview and current prices of them all the time. But you can also view the detailed data without having to add the stock to your favourites.

>So I can search for something, select it, then I go back to the regular view look at the chart.
That shouldn't be possible, because in compact view I don't think anything in search can be selected. After all, what would be the point of allowing that -- the user can't see anything change.